gssapi/raw/python_gssapi_ext.h

```diff
@@ -1,5 +1,43 @@
 #ifdef OSX_HAS_GSS_FRAMEWORK
 #include <GSS/GSS.h>
+
+// GSS includes these in gssapi_private.h but these aren't present on the host
```

Comment:
So far we've been loosely conforming to [krb5 style](https://k5wiki.kerberos.org/wiki/Coding_style/Formatting).  Because I find it easier to explain that visually, this first function would become:

```C
/* GSS includes these in gssapi_private.h but these aren't present on the
 * host, so just explicitly define them. */
GSSAPI_LIB_FUNCTION OM_uint32 GSSAPI_LIB_CALL
__ApplePrivate_gss_unwrap_iov(__nonnull OM_uint32 *minor_state,
                              __nonnull gss_ctx_id_t context_handle,
                              __nullable int *conf_state,
                              __nullable gss_qop_t *qop_state,
                              __nonnull gss_iov_buffer_desc *iov,
                              int iov_count);
```

(note that's all spaces, not tabs).

Reply:
I don't mind whatever way, this is literally a copy of the headers from https://opensource.apple.com/source/Heimdal/Heimdal-597.121.1/lib/gssapi/gssapi/gssapi_private.h.auto.html but with the names renamed. The renaming is part of the build process for Apple, they just don't include this header :(

Reply:
So far we've been converting to match project style in the cython, so I would prefer we keep doing that.  If you want to mention more detail about where these declarations come from in the comment here I think that would be fine - up to you.

Reply:
Sounds good, will do